test/data/overseas-passports-responses-and-expected-results.yml

```diff
@@ -752,7 +752,7 @@
 - :current_node: :which_country_are_you_in?
   :responses:
   - yemen
-  :next_node: :cannot_apply
+  :next_node: :apply_in_neighbouring_country
```

Comment:
I can see that this change was caused by this commit: 9c71c568436ad3bf4a5010104083d204b3f4d622 as part of #2463. So I'm happy to include it in this PR.